contracts/linear/src/validator_pool.rs

```diff
@@ -636,6 +736,13 @@ impl VValidator {
             VValidator::Current(v) => v,
         }
     }
+
+    pub fn get_weight(&self) -> u16 {
+        match self {
+            VValidator::V0(v) => v.weight,
```

Comment:
what about rename `VValidator` -> `VersionedValidator`?

Reply:
That's clearer definitely. But it's independent of this PR. I make another PR to fix this!
https://github.com/linear-protocol/LiNEAR/pull/141